### PR TITLE
Added lastSelectedInterventionActivity and lastSelectedInterventionEvent to OCKCareCardViewController

### PR DESCRIPTION
**Pull request by ninoguba**
This is similar to lastSelectedAssessmentEvent available in OCKSymptomTrackerViewController

### CODE REVIEW

File: CareKit/CareCard/OCKCareCardViewController.h [hunk 1]
```diff
@@ -130,6 +130,19 @@ OCK_CLASS_AVAILABLE
  */
 @property (nonatomic, weak, nullable) id<OCKCareCardViewControllerDelegate> delegate;
 
+/**
+ The last intervention activity selected by the user.
+ 
+ This value is nil if no intervention has been selected yet.
```
Comment (YuanZhu-apple) [inline CareKit/CareCard/OCKCareCardViewController.h]: if no `intervention activity` has been ...

File: CareKit/CareCard/OCKCareCardViewController.h [hunk 2]
```diff
@@ -130,6 +130,19 @@ OCK_CLASS_AVAILABLE
...
+ */
+@property (nonatomic, readonly, nullable) OCKCarePlanActivity *lastSelectedInterventionActivity;
+
+/**
+ The last intervention event selected by the user.
+ 
+ This value is nil if no intervention has been selected yet.
```
Comment (YuanZhu-apple) [inline CareKit/CareCard/OCKCareCardViewController.h]: if no `intervention event` has been ...

Comment (ninoguba) [inline CareKit/CareCard/OCKCareCardViewController.h]: Thanks.  I just updated the two lines of description...

**Comment by umerkhan-apple** [comment]
@ninoguba r=m3! 👍

**Pull request merged**